sdk/keyvault/azure-keyvault-secrets-apis/azure/keyvault/secrets/models.pyi

```diff
@@ -0,0 +1,63 @@
+# Stubs for azure.keyvault.secrets.models (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from ._shared import parse_vault_id
+from ._shared._generated.v7_0 import models as _models
+from datetime import datetime
+from typing import Any, Dict, Optional
+
+class SecretProperties:
+    def __init__(self, attributes: _models.SecretAttributes, vault_id: str, **kwargs: Any) -> None: ...
```

Comment:
`_models.SecretAttributes` is only an output type, correct?

Reply:
Correct. In fact all the models are for output only, users shouldn't construct them.